packages/node/test/machine/integration/install-virtual.spec.ts

```diff
@@ -242,7 +242,8 @@ describe("Scenario: Install virtual app with ERC20, put on-chain", () => {
         beneficiaries,
         capitalProvided: parseEther("10"),
         expiryBlock: (await provider.getBlockNumber()) + 1000,
-        tokenAddress: erc20Contract.address
+        tokenAddress: erc20Contract.address,
+        timeLockedPassThroughIdentityHash: HashZero
```

Comment:
So this test uses a commitment to the virtual app instance directly, without wrapping it in a TLPC contract... we should note this in a comment in the file